scripts/deploy-on-kind.sh

```diff
@@ -138,21 +138,10 @@ function deploy_korifi() {
     helm dependency update helm/korifi
 
     doDebug="false"
-    secLevel="restricted"
     if [[ -n "${debug}" ]]; then
       doDebug="true"
-      secLevel="privileged"
     fi
 
-    cat <<EOF | kubectl apply -f -
-apiVersion: v1
-kind: Namespace
-metadata:
-  labels:
-    pod-security.kubernetes.io/enforce: $secLevel
-  name: korifi
-EOF
-
     if [[ -n "$use_custom_registry" ]]; then
       helm upgrade --install korifi helm/korifi \
         --namespace korifi \
```

Comment:
We need to add the `--create-namespace` flag if we will no longer be creating the korifi namespace as part of the deploy script. In this case we probably need to patch the created namespace with the pod security standards labels, because helm won't do it. Alternatively this can become a manual step as well.

Reply:
oh yeah my mistake - I thought I was just moving the root namespace creation, but this is actually the Korifi namespace 😓 `--create-namespace` should be enough, I don't think we need to add the annotations

Reply:
Sounds good